crates/forge/src/main.rs

```diff
@@ -117,65 +118,67 @@ fn main_execution() -> Result<bool> {
         clean_cache(package_root).context("Failed to clean snforge cache")?;
     }
 
-    let mut all_failed_tests = vec![];
-    for package in &packages {
-        let forge_config = config_from_scarb_for_package(&scarb_metadata, &package.id)?;
-        let (package_path, package_source_dir_path) =
-            paths_for_package(&scarb_metadata, &package.id)?;
-        env::set_current_dir(package_path.clone())?;
-
-        // TODO(#671)
-        let target_dir = target_dir_for_package(&scarb_metadata.workspace.root)?;
-
-        let build_output = Command::new("scarb")
-            .arg("build")
-            .stderr(Stdio::inherit())
-            .stdout(Stdio::inherit())
-            .output()
-            .context("Failed to build contracts with Scarb")?;
-        if !build_output.status.success() {
-            bail!("Scarb build did not succeed")
+    let rt = Runtime::new()?;
+    let all_failed_tests = rt.block_on(async {
```

Comment:
This doesn't cause parallel execution, does it?

Reply:
cc @MaksymilianDemitraszek @mkaput

Reply:
https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#method.block_on

I'm not reading the code, but if the async block here is doing awaits, joins, whatever, then Tokio will parallelize this.